catalog/pages/inputs/index.md

```diff
@@ -973,7 +973,81 @@ span: 6
               </Select>
             </Column>
         </Row>
-        <Spacing top={{small: "normal"}} />
+    </Container>
+</ThemeProvider>
+```
+
+## Hybrid Select
+
+### Props
+
+```table
+span: 6
+rows:
+  - Prop: value
+    Type: array
+    Default: "[ ]"
+    Notes: Specifies array of initial string values
+  - Prop: onChange
+    Type: function
+    Default: null
+    Notes: Invoked with an array of updatedSelection when a option is selected by the user
```

Comment:
Grammatical mistake it should be:  Invoked with an array of updatedSelection when **an** option is selected by the user

Reply:
Updated.